cmd/utils/nodecmd/flags_test.go

```diff
@@ -68,7 +68,7 @@ var flagsWithValues = []struct {
 	{
 		flag:        "--dbtype",
 		flagType:    FlagTypeArgument,
-		values:      []string{"leveldb", "badger"},
+		values:      []string{"LevelDB", "BadgerDB", "MemoryDB"},
```

Comment:
Is there any reason to change this?

Reply:
dbtype should have the same values of `DBType`. If I changed `DBType` to "leveldb" or "badger", the log would not print pretty.
```
const (
	LevelDB   DBType = "LevelDB"
	BadgerDB         = "BadgerDB"
	MemoryDB         = "MemoryDB"
	DynamoDB         = "DynamoDB"
	ShardedDB        = "ShardedDB"
)
```